subsys/bluetooth/controller/Kconfig

```diff
@@ -6,7 +6,11 @@
 # The following symbols are enabled depending if the controller actually
 # supports the respective features.
 
+config BT_CTLR_ENTROPY_SUPPORT
+	bool
```

Comment:
Would you mind adding a help text to describe what this means? Most other "_SUPPORT" entries are for Bluetooth features defined in the spec. I assume this one is describing if the HCI_LE_Rand function is available?
From the spec   HCI_LE_Rand  is mandatory if encryption is available, so maybe we can reuse BT_CTLR_LE_ENC_SUPPORT ?

Reply:
Can we move this comment to the Kconfig's help text?

Reply:
Seems like the help text for BT_CTLR_ENTROPY_SUPPORT got lost :)